### ISSUE TITLE
WHERE IS WEEK 7. THE SOURCE CODE IS RELEASED

### ISSUE THREAD

**Issue by VannyTheBunny09** [luckydog7/Funkin-android#853]
BRUH. JUST DO IT

**Comment by ROBIKALIVE13** [comment]
> BRUH. JUST DO IT

JUST WAIT

**Comment by netonero** [comment]
Oh my fucking god now i know how people are impatient as fuck

**Comment by pump776** [comment]
then you can make a week 7??

**Comment by kffwlp** [comment]
Please be patient. I'm sure @luckydog7 is porting it

**Comment by pump776** [comment]
> Please be patient. I'm sure @luckydog7 is porting it

I hope @luckydog7 can do week 7 I can't wait to play week 7 with cutscenes

**Comment by AnthonyCTM** [comment]
Yeah just wait, luckydog7 already made a cutscene test and looks like it is working flawlesly so it's only matter of time to have the week 7

**Comment by pump776** [comment]
> Yeah just wait, luckydog7 already made a cutscene test and looks like it is working flawlesly so it's only matter of time to have the week 7

good news.after release I will immediately play it

**Comment by AnthonyCTM** [comment]
But i think that the week 7 may be the last update related to mayor changes cuz ninjamuffin reached the goal to make an official mobile port of the game, but that are only my thoughts

**Comment by bloodheroC** [comment]
1 day left in the Kickstarter

**Comment by bloodheroC** [comment]
Im excited

**Comment by Noahgboy** [comment]
34 mind left!!!!!!

**Comment by Noahgboy** [comment]
But it's gonna come in 2022

**Comment by pump776** [comment]
> But i think that the week 7 may be the last update related to mayor changes cuz ninjamuffin reached the goal to make an official mobile port of the game, but that are only my though

Seriously??

**Comment by ItzLiquid** [comment]
> BRUH. JUST DO IT

Is this a reference to "JUST DO IT"?
